docs/docs/manual/wikidata.md

```diff
@@ -6,28 +6,251 @@ sidebar_label: Wikidata
+*   P17|P749/P17
+*   P17|(P749/P17)
+*   (.|P749)/P17
+
+They fetch the [country (P17)](https://www.wikidata.org/wiki/Property:P17) of an item or that of its [parent organization (P749)](https://www.wikidata.org/wiki/Property:P17).
+
+### Terms: labels, descriptions, aliases, and sitelinks
+
+Entities on Wikidata have "terms:" labels, descriptions, aliases, and sitelinks. Each entity has human-readable preferred names (labels) in one or more languages. Each label comes with a description field, and a space for alternative labels in that language. Entities also can have a link to the related Wikipedia page in one or more langauges. 
+
+For example, [Q5](https://www.wikidata.org/wiki/Q5) has the English label "human" and the aliases "person" and "people," with a description including "common name of Homo sapiens;" it also has the Italian label "umano" and the alias "persona" with a description "specie a cui appartiene il genere umano." The English sitelink is "https://en&#46;wikipedia.org/wiki/Human" but Italian Wikipedia has no equivalent article attached to Q5. Not every language will have values, but these terms can still be useful for reconciling. 
+
+After reconciliation, you can extend your data with these terms. You can fetch aliases for a given term, or labels in another languages, by using <span class="menuItems">Edit column</span> → <span class="menuItems">Add columns from reconciled values....</span>. 
+
+You can refer to a term in a specific language by manually entering in a three-letter code. The first letter is the term ("L" for label, "D" for description, "A" for aliases, "S" for sitelink) and the next two are the language code. For example:
+
+*   `Len` for Label in English
+*   `Dfi` for Description in Finnish
+*   `Apt` for Alias in Portuguese
+*   `Sde` for Sitelink in German
+
+Enter in the three-letter code manually in the <span class="menuItems">Add Property</span> field. You can manually enter as many term/language combinations as you require. 
+
+No language fall-back is performed when retrieving the values.
+
+### Comparing values
+
+By default, data in OpenRefine and entities in Wikidata are compared by string fuzzy-matching. There are some exceptions to this:
+*   If the value is an identifier, then it confirms the exact string without matching.
+*   If the values are integers, exact equality between integers is used.
+*   If the values are floating point numbers, the score is 100 if they are equal and decreases towards 0 as their absolute difference increases.
+*   If the values are coordinates (specified in the "lat,lng" format on OpenRefine's side), then the matching score is 100 when they are equal and decreases as their distance increases. Currently a score of 0 is reached when the points are 1 kilometre away from each other.
+
+Sometimes, we need a more specific matching on sub-parts of these values. It is possible to select these parts for matching by appending a modifier at the end of the property path:
```

Comment:
This is really useful information but I wonder if it should be in the OpenRefine docs or whether we should have a pointer to external documentation on the Wikidata reconciliation service?

Reply:
That makes sense, it should be documented externally indeed.

Reply:
Do you mean drop the text into someplace like https://www.wikidata.org/wiki/Wikidata:Tools/OpenRefine#Wikidata_reconciliation? It doesn't have much space to explicate, at present.

Or somewhere like https://wikidata.reconci.link/? That just points back to our wiki, and to https://github.com/wetneb/openrefine-wikibase - which is another potential place to document it.

Reply:
Sorry, I should've started with: I can't find this documented anywhere else on the web except in our own wiki.

Reply:
Yes it should be documented in https://github.com/wetneb/openrefine-wikibase, or perhaps on the web service itself, https://wikidata.reconci.link/

Reply:
@wetneb has put a version of this information here: https://wikidata.reconci.link/#documentation

It looks like it's just copied from the original wiki text. I have an updated version of the same text, but I've now removed it from this draft and saved it elsewhere. If we'd like to update the text on that page, we can take care of it another time.